lib/nx_signal.ex

```diff
@@ -39,19 +40,20 @@ defmodule NxSignal do
       >
       iex> f
       #Nx.Tensor<
-        f64[frequencies: 2]
+        f32[frequencies: 2]
         [0.0, 200.0]
       >
   """
   defn stft(data, window, opts \\ []) do
     {frame_size} = Nx.shape(window)
 
-    {overlap_size, fs} =
+    {overlap_size, fs, padding} =
       transform({frame_size, opts}, fn {frame_size, opts} ->
```

Comment:
Consider implement `stft` as `deftransform` and passing `fs` as an argument to `defn`, so it is properly converted to a tensor!